sync-superchain.sh

```diff
@@ -50,8 +50,8 @@ process_network_dir() {
         chain_id=$(dasel -f "$toml_file" -r toml "chain_id" | tr -d '"')
         chain_name="$(basename "${toml_file%.*}")"
 
-        # Skip if chain_id is empty or Boba's (28882)
-        if [[ -z "$chain_id" || "$chain_id" -eq 28882 ]]; then
+        # Skip if chain_id is empty or Boba Sepolia (28882) or Boba Mainnet (288)
+        if [[ -z "$chain_id" || "$chain_id" -eq 28882 || "$chain_id" -eq 288 ]]; then
```

Comment:
There is an exception for boba mainnet in the superchain-registry for genesis-integrity-check so I mirrored that exception ehre